File: mini-package-api/mini-package-api/Models/Inquire.cs
using mini_package_api.Enums;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace mini_package_api.Models
{
    public class Inquire
    {
        public int InquireId { get; set; }

        public Guid InquireGuid { get; set; }

        public decimal PackageSizeWidth { get; set; }

        public decimal PackageSizeHeight { get; set; }

        public decimal PackageSizeLength { get; set; }

        public decimal Weight { get; set; }

        public DateTime CreatedDate { get; set; }

        public DateTime PickupDate { get; set; }

        public DateTime DeliveryDate { get; set; }

        public int DestinationClientAddressId { get; set; }

        public virtual ClientAddress DestinationClientAddress { get; set; }

        public int SourceClientAddressId { get; set; }

        public virtual ClientAddress SourceClientAddress { get; set; }

        public PriorityEnum Priority { get; set; }

        public bool DeliveryAtWeekend { get; set; }

        public bool IsIncoming { get; set; }

        public virtual Offer Offer { get; set; }

        public decimal DisplayPrice { get; set; }
    }
}

File: mini-package-gui/src/app/models/inquire.ts
import { Address } from "./address";

export class Inquire {
  clientId: number;
  inquireId: number;
  packageSizeWidth: number;
  packageSizeHeight: number;
  packageSizeLength: number;
  weight: number;
  createdDate: Date;
  pickupDate: Date;
  deliveryDate: Date;
  destinationAddress: Address | null;
  sourceAddress: Address | null;
  priority: string;
  deliveryAtWeekend: boolean;
  isIncoming: boolean;
    
  constructor() {
    this.clientId = 0;
    this.inquireId = 0;
    this.packageSizeWidth = 0;
    this.packageSizeHeight = 0;
    this.packageSizeLength = 0;
    this.weight = 0;
    this.createdDate = new Date();
    this.pickupDate = new Date();
    this.deliveryDate = new Date();
    this.destinationAddress = null;
    this.sourceAddress = null;
    this.priority = "Low"
    this.deliveryAtWeekend = false;
    this.isIncoming = true;
    }
}
